package.json

```diff
@@ -10,9 +10,10 @@
   },
   "dependencies": {
     "@gravity-ui/axios-wrapper": "^1.3.0",
+    "@gravity-ui/components": "^2.9.1",
```

Comment:
Added new package to dependencies, since the embedded ui is also used as a package itself